packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -37,7 +37,15 @@ const generationConfig = {
   "disableGen": false
 }
 
-export const chooseDatabase = async(): Promise<string> => {
+const paths = {
+  "model": "model",
```

Comment:
```suggestion
  "model": "/model",
```

Reply:
thank Ankit. I'm going to close this PR as there's a slightly different approach to this that I want to try out following another format of providing paths with : `./something`
Thanks for the valuable feedback!